### PR TITLE
add yukari new method

### PR DESCRIPTION
**Pull request by mapleincode**

### CODE REVIEW

File: lib/yukari.js [hunk 1]
```diff
@@ -525,6 +525,33 @@ Yukari.prototype.update = function(callback) {
 };
 
 /**
+ * update  by json
+ * @data jsonData
+ */
```
Comment (XadillaX) [inline lib/yukari.js]: /**
 * update by JSON
 * @param {Object} jsonData the json data to be updated
 * @param {Function} callback the callback function
 */

File: lib/yukari.js [hunk 2]
```diff
@@ -525,6 +525,33 @@ Yukari.prototype.update = function(callback) {
...
+
+Yukari.prototype.updateByJson = function(jsonData, callback) {
+    if ("string" === typeof jsonData) {
```
Comment (XadillaX) [inline lib/yukari.js]: 你确定不是 `"function" === typeof jsonData`？

**Comment by XadillaX** [comment]
review together. @luicfer @amunu

File: lib/yukari.js (hunk 2 above)
Comment (mapleincode) [inline lib/yukari.js]: _(:з」∠)_

Comment (luicfer) [inline lib/yukari.js]: O(∩_∩)O

**Comment by XadillaX** [comment]
LGTM. how about you? @luicfer @amunu

**Comment by XadillaX** [comment]
@mapleincode 能给这个 PR 写个测试吗？

**Comment by luicfer** [comment]
LGTM

**Pull request merged**